bitnami/argo-cd/templates/applicationset/deployment.yaml

```diff
@@ -0,0 +1,183 @@
+      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: applicationSet
+        {{- if .Values.applicationSet.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ include "argocd.applicationSet.serviceAccountName" . }}
+      {{- include "argocd.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.applicationSet.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.applicationSet.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.applicationSet.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.applicationSet.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.applicationSet.podAffinityPreset "component" "applicationSet" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.applicationSet.podAntiAffinityPreset "component" "applicationSet" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.applicationSet.nodeAffinityPreset.type "key" .Values.applicationSet.nodeAffinityPreset.key "values" .Values.applicationSet.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.applicationSet.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.applicationSet.priorityClassName }}
+      priorityClassName: {{ .Values.applicationSet.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.applicationSet.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.applicationSet.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: applicationset-controller
+          image: {{ include "argocd.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.applicationSet.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.applicationSet.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.args "context" $) | nindent 12 }}
+          {{- else }}
+          args:
+            - argocd-applicationset-controller
+            - --logformat
+            - {{ .Values.applicationSet.logFormat }}
```

Comment:
I would expect to quote these settings for consistency with the rest of the args